Contributing.md

```diff
@@ -166,19 +153,60 @@ Once your changes are successfully merged into the main repository, delete the b
 
 Congratulations, you have successfully contributed to the project!
 
-## How to use Markdown to format your topic
+## Adding samples
+
+Samples are a critical tool for learning the Office Scripts API. We welcome samples scripts from the community to help demonstrate useful scripting scenarios.
```

Comment:
"Samples scripts" reads a little odd to me. Would the sentence also be correct with "sample scripts"? 

```suggestion
Samples are a critical tool for learning the Office Scripts API. We welcome sample scripts from the community to help demonstrate useful scripting scenarios.
```